HS-Tools/HS-BG-Leaderboards-Bot
# feature(slash-commands)

updated from discord.py to interactions.py. This changes all the commands
to slash commands which are simpler and comply with updates to discords
terms of service.

TODO:
- Test with discord token
- Ensure embeds and other messages are set correctly
- Ensure that channel restrictions are set correctly
    - most likely need to add `bot.command(scope)`
    - most likely need to update default scope to just wall_lii
